package.json

```diff
@@ -91,6 +91,13 @@
       "pre-commit": "node automation/check-npm-version.js && node automation/check-doc.js"
     }
   },
+  "mocha": {
+    "reporter": "spec",
+    "require": "ts-node/register/transpile-only",
+    "file": "./tests/config-tests",
+    "timeout": 12000,
+    "_": "tests/**/*.spec.ts"
```

Comment:
These are just defaults, for instance this positional arg is still overridden in `test:only` above